coderd/coderdtest/coderdtest.go

```diff
@@ -134,6 +134,10 @@ type Options struct {
 	ConfigSSH codersdk.SSHConfigResponse
 
 	SwaggerEndpoint bool
+
+	// LoggerIgnoreErrors causes the test logger to not fatal the test on Fatal
+	// and not error the test on Error or Critical.
+	LoggerIgnoreErrors bool
```

Comment:
I'm not sure we should make this an option... if an error isn't a failure, we should change the log.

If I'm misunderstanding, please link the log in question so I can take a closer look 🕵️🕵️🕵️

Reply:
Yeah I was quite conflicted about this.  It implies to me that the logger's opinion is either that testing failures is an anti-pattern or that a workspace build job failing does not qualify to be logged as an error.

I believe it was this log here: https://github.com/coder/coder/blob/467b0a10046337570b9a7d21f6e4609c79ecf809/provisionerd/runner/runner.go#L929-L932

Maybe this should only be warning level?

Reply:
Ahh, yes. I'd say warning level is fine!